compiler/lib/source_map.ml

```diff
@@ -19,6 +19,16 @@
 
 open! Stdlib
 
+module Source_content = struct
```

Comment:
Nit: naming it `Sources_content` (with an S to Sources) would match the sourcemap field name `sourcesContent`.

Reply:
The sourcemap field store multiple sources but `Source_content.t` is about a single one.

Reply:
Ah, true. Never mind me then.